CHANGELOG.md

```diff
@@ -3,6 +3,20 @@
 *Important!*
 - The next release will be 1.0.0, which means that master will contain breaking changes.
 
+## 1.0.0.pre.1
+
+*Important!*
+
+- This is the final release of KubernetesDeploy. Version 1.0.0 will be released
+under the name `Krane`. We've added a migration guide to help make it easier to migrate.
+([#607](https://github.com/Shopify/kubernetes-deploy/pull/607))
+
+*Enhancements*
+- (beta) `krane deploy` will now consider all namespaced resources eligible for pruning,
```

Comment:
How many does this add? Should we make a note which ones this adds compared to our whitelist as of the time of writing?